### PR TITLE
Send email notifications about calendar events changes

### PR DESCRIPTION
**Pull request by sponkrashin**
Issue #354

### CODE REVIEW

File: server/Arcadia.Assistant.Server.Console/appsettings.json [hunk 1]
```diff
@@ -23,11 +23,33 @@
       "Port": 0,
       "UseTls": true
```
Comment (santee) [inline server/Arcadia.Assistant.Server.Console/appsettings.json]: Put 'Enabled' here, since you have it in configuration

Comment (sponkrashin) [reply server/Arcadia.Assistant.Server.Console/appsettings.json]: Done.

File: server/Arcadia.Assistant.Calendar.Notifications/EventAssignedToApproverNotificationActor.cs [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    using Arcadia.Assistant.Calendar.Abstractions;
+    using Arcadia.Assistant.Calendar.Abstractions.EventBus;
+    using Arcadia.Assistant.Configuration.Configuration;
+    using Arcadia.Assistant.Notifications;
+    using Arcadia.Assistant.Notifications.Email;
+    using Arcadia.Assistant.Organization.Abstractions;
+    using Arcadia.Assistant.Organization.Abstractions.OrganizationRequests;
+    using Arcadia.Assistant.UserPreferences;
+
+    public class EventAssignedToApproverNotificationActor : UntypedActor, ILogReceive
```
Comment (santee) [inline server/Arcadia.Assistant.Calendar.Notifications/EventAssignedToApproverNotificationActor.cs]: that's the thing, it's not really a general notification actor. It has to know all about email/push settings which is not really good. We either need to rename this one, or split it on two (push and emails) with one more layer between them

Comment (sponkrashin) [reply server/Arcadia.Assistant.Calendar.Notifications/EventAssignedToApproverNotificationActor.cs]: We already have common email and common push actors, we don't need more layers.
We can split this on `EventAssignedToApproverEmailNotificationActor` and `EventAssignedToApproverPushNotificationActor` and each actor responsibility will be to wrap and send notification of specific type.
Or we can assume that this concrete actor responsibility is to send notifications of all known types about event assigned to approver.
First is more verbose, but each actor has only one reason to change. Second is less verbose, but actor has several reasons to change. What do you prefer?

Comment (sponkrashin) [reply server/Arcadia.Assistant.Calendar.Notifications/EventAssignedToApproverNotificationActor.cs]: Done.

**Pull request merged**